utils/check_repo.py

```diff
@@ -413,22 +413,15 @@ def get_model_modules() -> List[str]:
         "modeling_auto",
         "modeling_encoder_decoder",
         "modeling_marian",
-        "modeling_mmbt",
-        "modeling_outputs",
```

Comment:
we've changed the model structure since 4 years ago, when this list was first created: `transformers.models` exclusively contains folders with models. 

As such, we don't need to manually ignore these files.